src/Agent.Plugins/PipelineArtifact/PipelineArtifactServer.cs

```diff
@@ -129,9 +124,9 @@ public class PipelineArtifactServer
                     throw new InvalidOperationException("Unreachable code!");
                 }
 
-                await DownloadPipelineArtifact(
+                await DownloadPipelineArtifacts(
                     buildDropManager, 
-                    buildArtifact, 
+                    new List<BuildArtifact>().Append(buildArtifact),
```

Comment:
what about `new List<BuildArtifact>(){buildArtifact},`?

Reply:
Sure. I don't see why not.